x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/__test__/__snapshots__/DetailView.test.js.snap

```diff
@@ -906,7 +906,7 @@ exports[`DetailView should render with data 1`] = `
                 >
                   if
                 </span>
-                 (req.paarms.level === 
+                 (req.params.level === 
```

Comment:
This is an example of a spelling error, which will cause a `TypeError`. So shouldn't be fixed.